issue_template.md

```diff
@@ -1,3 +1,10 @@
+#
+# STOP! ✋
+#
+# DO NOT OPEN A NEW ISSUE!
```

Comment:
maybe we could add something like "There's a new and better way to raise your question"??
"DO NOT OPEN A NEW ISSUE!" sounds a bit harsh by itself.

What do you think @DrTorte?

Reply:
I think we can add that, underneath, but I think that having this big text is important.